l10n_es_aeat_mod349/views/mod349_view.xml

```diff
@@ -188,6 +188,12 @@
                 <button name="%(l10n_es_aeat.action_wizard_aeat_export)d" position="attributes">
                     <attribute name="name">%(action_wizard_aeat_mod349_export)d</attribute>
                 </button>
+                <field name="period_type" position="replace">
```

Comment:
Tks!!!
No es por feo, es por tener la relación entre los dos campos más clara.

2016-06-03 13:27 GMT+02:00 Pedro M. Baeza <notifications@github.com>:

> In l10n_es_aeat_mod349/views/mod349_view.xml
> <https://github.com/OCA/l10n-spain/pull/367#discussion_r65693050>:
>
> > @@ -188,6 +188,12 @@
> >                  <button name="%(l10n_es_aeat.action_wizard_aeat_export)d" position="attributes">
> >                      <attribute name="name">%(action_wizard_aeat_mod349_export)d</attribute>
> >                  </button>
> > +                <field name="period_type" position="replace">
>
> Bueno, lo dejo por esta vez por no bloquear esto mucho, pero sigo pensando
> que no queda tan feo teniendo el check en la siguiente línea haciendo un
> position="after"
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OCA/l10n-spain/pull/367/files/df8b8a5d94757ae1a5a4332bf4a9860aec3feef7#r65693050>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKpVCySTi-zmA7L2VwOajZ1tfqE_hQnMks5qIA-PgaJpZM4ItbTI>
> .
>



-- 
=========================
Santi Argüeso
Twitter: @santiarg

Web: http://comunitea.com
Twitter: @comutuitea
Tfno: 982814805